### ISSUE TITLE
Docs not expanding template literal for latest version number in CDN urls for no bundler example

### ISSUE THREAD

**Issue by 121yaseen** [Semantic-Org/Semantic-UI-React#4185]
### Steps
Add CSS for Semantic UI using CDN option ie, using the [code snippet](https://react.semantic-ui.com/usage#option-2-cdn-no-bundler) given in  the official page.

### Expected Result
Add CSS without any error

### Actual Result
The request was returned with 404 error `GET https://cdn.jsdelivr.net/npm/semantic-ui@$%7Bprops.versions.sui%7D/dist/semantic.min.js net::ERR_ABORTED 404` 💣

### Version
v2.0.3

**Comment by brianespinosa** [comment]
@121yaseen it looks like our docs need an update. That template literal should expand to match the latest version of SUIR. Once you change it to the latest version, it should work for you:

```
//cdn.jsdelivr.net/npm/semantic-ui@2.0.3/dist/semantic.min.js
```

**Comment by layershifter** [comment]
@121yaseen thanks for reporting, I overlooked it in #3713. Will be fixed by #4221.

**Comment by layershifter** [comment]
Was fixed in [`semantic-ui-react@2.0.4`](https://github.com/Semantic-Org/Semantic-UI-React/releases/tag/v2.0.4).
